greenplumpython/op.py

```diff
@@ -0,0 +1,11 @@
+from typing import Callable
+
+from .db import Database
+from .expr import BinaryExpr, Expr
+
+
+def operator(name: str, db: Database) -> Callable[[Expr, Expr], BinaryExpr]:
+    def make_operator_expr(left: Expr, right: Expr) -> BinaryExpr:
+        return BinaryExpr(name, left, right, db=db)
```

Comment:
```
left_type
    The data type of the operator's left operand, if any. This option would be omitted for a prefix operator.
```

https://www.postgresql.org/docs/current/sql-createoperator.html

Shall we support **prefix operator**?

Reply:
Based on my current knowledge on operators, I don't think we should support prefix operators here.

AFAIK, the primary use case for operators is Index Scan, where the operators are always binary ones. It is very rare that the user needs to create prefix operators for analytics. We should also not encourage them to do so becuase operators are much less readable than functions. 

Therefore, I think in most cases, the three built-in prefix operators, i.e. +, -, and NOT, should be enough. This is why I don't implement `create_operator()` here.